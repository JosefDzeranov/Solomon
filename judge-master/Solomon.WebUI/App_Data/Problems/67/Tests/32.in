939
439507742 213446938 380335
184425450 684600460 794336
932546276 61126610 496144
367181570 175957970 916868
94015596 382950902 414481
255009345 41802448 448740
939957806 15832318 457685
286329998 354098084 687868
20203910 6750652 960669
757878097 34188974 969335
735100857 80613481 754554
386270767 372298904 164103
115100678 336516192 948279
632039949 297486417 504005
201893942 81063275 411113
552439977 206809070 979129
758687669 134791970 536840
812815858 83233788 5978
653982071 229459155 932406
582845655 154747651 981731
739794860 229776925 629830
366767784 491986710 561409
935889782 44233335 976220
645242566 10292717 70061
110779881 425790512 946569
546909160 100770081 462540
439017954 353421503 363509
107593223 364513254 407640
85820272 621290503 72807
871017166 107786421 682493
22234374 222110017 159875
394006183 428120845 711755
722481143 38434311 363148
409863301 199183895 235185
832435878 81568841 72479
48948619 345791599 902731
418623682 424594871 691479
840869766 112684422 907235
653741528 62634419 409871
430168408 469824851 25587
943033213 46223045 346495
545289592 191189472 168778
986539511 11810618 107906
395831938 452706481 139092
361129587 211137909 540318
288713210 432358087 966946
583811625 261455978 849956
957028748 414102 628172
31595498 634047767 557588
674697009 172788937 131030
560776561 6109146 909198
245600139 231396063 160506
895509247 3928632 884177
148506653 820492345 691780
41822723 841926980 36625
15892812 80651848 691295
317271742 235902834 296096
617687937 125888741 326155
183839631 642536226 370463
325386872 99354970 952892
903824305 15010657 786080
726314296 74039712 390253
770094615 69517789 426663
37603661 313631587 724348
625575683 251152551 486010
195914453 98384370 345606
21110270 829682538 449222
496620667 180033651 394860
784496498 36577431 268319
876159800 24588126 216186
808156662 67929793 100556
213396380 582595033 556124
546756325 261656128 341337
929701424 33422667 241052
735947551 228902346 412553
558838521 318218471 601702
146224831 401242872 322147
115318131 352457576 735037
977694867 19778795 9718
683188860 4750822 853505
222679497 430848644 955288
755552323 184118405 623258
629908390 324437253 653057
293368277 338414888 964298
553980220 335026843 175165
543938335 393409709 555053
43556073 11829161 642558
879776176 119948451 639846
626753205 246436480 326812
880205980 3588940 330290
507653246 284714864 237399
995241070 2612516 613707
60800652 222889664 237025
412229600 483980338 787398
911283069 39758932 985578
340085352 438126174 954701
832531761 7744785 637510
910706393 5028162 152347
773962924 57592342 191017
227985218 348301432 439970
954304921 10068942 129649
904560625 64402644 37606
949213147 49301724 436693
797407085 89934824 233410
74002072 875902720 174227
204930519 40376622 572896
325184672 317374855 269888
591877310 338563162 219239
746059617 48480380 37674
986207719 8908850 618721
594149265 279461535 823962
104985695 872027443 478626
790898960 185203188 359022
6916525 917197868 882622
222656996 283249887 301727
506996456 125950892 853744
849268018 135227189 32830
834357719 40698500 390846
984597639 7277835 284805
19894035 723322382 25876
896123071 7176644 147939
12134618 945749901 900848
719821602 199641776 827643
764600672 170068783 682463
921614383 57450228 48070
490120727 463726379 323888
60148790 624877314 524690
84765879 858021886 970062
469999453 4582477 513794
209367522 613031531 442373
267915233 496186821 876058
420967221 178966081 303116
232281945 392525984 773863
159577777 819874278 980838
923554446 38832847 235297
493366805 412075817 570622
904201512 80996842 106164
68478667 121680496 94174
347088844 419989720 350304
783117338 41007759 63029
653410505 11202084 673780
284019285 276056813 365003
697876620 237219596 558003
546726597 240692641 53074
323627440 226428096 88988
850475804 104227422 425218
112484251 541761959 972439
937860673 12087302 556114
927158639 54115948 731984
427040977 565095176 721490
371161027 132060937 86188
526242904 458475144 738010
484431888 55021181 677068
315767115 180202677 154309
672299570 133879981 569466
390046669 594665374 608533
795000445 160707210 349832
185040902 172574357 397451
550274974 29980030 203571
756337127 133692576 728809
94807588 901334661 133764
277188147 449512043 562985
574029500 366524005 903815
909291244 42266067 616461
598342130 380206608 946799
422722356 176972669 456966
216957743 333702080 420337
840918388 81549283 621671
556260137 239737570 116373
320863830 142448626 743067
733621728 116712703 336971
120740194 471000828 909619
257058104 677556701 665394
450426175 11677185 170811
492823893 398941507 745576
78276848 546248256 106865
274240233 337935163 509588
45361593 426573239 857768
273814964 53471514 455699
623474895 47970704 304005
968827854 8738317 37618
836809385 88928068 192501
266056663 106903372 194998
252401986 72480153 668058
52405296 670007042 719883
976687236 2964713 900977
597572964 48503142 21798
750853637 81993995 544627
678353517 97383944 954720
514383072 55712841 903402
729821494 14183941 920532
524493214 6386902 412158
943894495 41311728 391802
107353711 133336095 787624
711937932 45620497 153831
153275356 540140019 532682
543918041 334346201 79
880242354 8333615 269398
732435912 34117663 933679
236444403 631083500 358524
571202966 142158443 402489
803496959 75561778 75381
375760094 422759237 760019
126014067 702812792 345619
735469711 95279002 35698
739959964 22164441 973187
859008260 19311235 60325
234592548 229326776 378786
217434583 493074735 405591
835272942 157293585 194119
513102260 69894576 249501
639432954 36128056 589392
347278763 151375055 873458
525874987 238456570 815382
256595552 466938162 581354
38023249 59772046 689257
683094417 125164290 156829
869882679 15177895 740086
448615312 375979953 243197
640889132 155173050 970140
40226808 122377242 27739
547721863 396396994 41052
690493280 299594104 778458
681618284 107302647 533435
485901638 428654906 112756
676755443 238380250 808109
462723384 361635466 725501
210391757 547533300 42228
985028571 2400652 689521
684902660 156909126 110552
404537210 382734851 515229
310728524 362170563 392940
23619080 536264951 97194
517157927 315397092 138288
878262696 8952006 498474
924731667 67446998 655220
239040496 662365064 559065
234914501 603610649 567534
146450701 305553252 84243
568115689 244278928 613680
3842955 143478489 672575
447132186 64596139 790602
650224260 234372241 587985
111165821 860368290 849147
580655393 25876466 650185
539702660 191358250 94080
925122582 20891279 56506
529831525 322988239 794469
428923590 549598416 968825
329473848 96902566 276040
844122927 129857899 928671
467555635 37175351 676079
335318797 311338307 443056
441702882 416042535 110743
720246713 270436121 467405
883481469 8876447 19159
585584022 389774361 214872
744650126 69858383 959084
236306930 172926472 316234
435815111 149660905 400242
309054292 518167525 133557
791231328 79746980 954774
940538504 2149740 388823
722785640 78749971 980049
322004279 116382539 462342
554656589 386603308 888674
767258579 104089043 718346
223970923 695433688 392755
319728615 7663682 244158
154504167 26644032 889190
753615559 237471907 625092
285102640 106811934 912591
493294381 64528545 713487
346334799 491947791 902
623496194 10643514 46876
167444593 284924406 677267
360185266 416878265 816307
270835721 384534492 332052
89991114 734595791 213207
881517237 59425371 190149
209568224 722666031 81820
985692603 1798926 575747
240869445 650657413 431900
584953306 58487105 610015
1348532 840748286 125367
482591943 254437350 954490
190676552 200485182 609465
610178070 247373126 799852
58398364 630510845 76451
59865685 178889751 974856
141293599 479581827 339644
276761849 434423658 787813
377766739 165189845 478595
118743911 210121643 958184
849791390 86082103 465427
656417361 149944108 279519
516151810 136957461 736692
775486089 119825263 178424
11226853 893240577 362308
87172656 366065551 498408
121502738 620297501 761780
303628712 442286696 656474
742202322 60614217 473952
228632418 11090479 528991
376175398 336656534 658970
455710730 320822828 504228
461912718 245086881 107339
537000166 121453432 775831
513096331 403808146 350121
854908207 64453599 188404
795480698 181651856 352731
779173866 7523167 31010
533664761 160729376 941864
953190544 11502431 542274
455985374 420634958 271718
256412839 319123255 976763
155587096 537587000 670317
383602005 541397338 854518
828323648 35855033 243903
316281998 546325211 683062
831485976 65656075 3449
978268699 21711731 742780
838557220 97662103 975374
371497656 292321414 573471
764701706 229747071 162613
141468765 322085318 199432
776562396 35122770 373171
140848050 519189087 539864
540969495 306392866 985653
819252204 5555779 139729
709539337 160041963 569385
189049119 545336987 28946
222496770 254682345 691869
684049465 267178758 88815
679712940 266680666 851094
719839630 20021874 266949
535350619 433109369 226331
675351904 156245776 191561
973558388 3593447 856502
968088955 26861731 287629
678425127 208886606 593429
77446464 211062756 584076
342410882 522617111 860313
440084767 193122566 547981
374079209 217062135 337711
763605510 50623044 531042
480064065 150007548 628586
753373334 61088609 337370
677567028 100611273 677214
95140925 753615157 907832
912086758 37253369 390539
102003629 525134418 683493
420165741 387506159 145583
361411222 562426777 517229
283643498 88919913 945327
651341128 210507303 970380
316533349 102360118 955682
518163863 179298062 995129
886977742 59201642 343857
969663366 22343555 700764
59567617 423856175 222539
543139368 199029759 588461
573750195 168861201 122328
917971276 77760977 490145
383182428 26051345 928808
933104011 36113108 330339
651559534 345482536 446208
909667621 58633261 919670
724098400 200084320 672975
754173101 157146164 496457
118810075 334869291 954198
295552610 676165629 72816
460129514 269575979 674935
516395226 307724795 589036
601510314 228678735 995427
907437507 63600569 116516
282819961 92015480 672232
533916118 245794924 124740
386446131 386495118 841480
719800827 146859730 19622
811189365 109218021 975811
367771843 114426159 890523
8209199 445357828 50278
11499188 28000788 568393
143003820 164878858 141847
803792093 99142641 596207
127087052 746144077 873107
572679193 203443324 393460
870650705 68858695 196320
874207218 31065898 84776
320960821 386073284 796617
456132015 34757439 128769
554056428 53892899 757301
453539528 193539073 56065
499713999 474220975 587674
86464858 308024895 813985
9775873 3997745 484984
139898440 351962979 461269
160395904 806991042 974252
332795372 211315931 362879
741885772 177671487 834290
35471904 886016123 167367
43890352 552848997 738105
563535145 339982339 995869
174329501 218062732 647818
938103342 53605913 875728
394719400 416433746 429821
67819378 322813330 142179
345366223 336118200 577489
693040804 80687640 931016
456639011 378815759 357422
735216477 38662318 579830
17610919 834515523 984792
103025579 837309577 210055
474577013 292475409 865112
80577897 411002839 93089
668776666 281130681 656911
803403533 29170478 984448
906134315 14968012 647411
721598130 179500724 802198
935524331 49069555 958184
166652420 105501402 720522
172165427 456002679 948341
6668907 914065927 601314
748216001 218342614 588190
84744982 202924243 484416
54040788 186756916 847515
396351845 158779087 693503
884229653 105408109 122846
897159791 59956978 416769
731113742 6203054 998546
924319780 7170710 105625
62126500 218111047 234913
634784589 267824604 377627
251176937 326211322 292194
623877903 368173827 973438
91475230 465092307 914355
488840175 219466611 924014
263315474 159985789 128123
773698092 47908080 422169
473250217 155638650 821317
267055841 653453721 332246
102765530 56814242 464569
520935174 206269185 345000
317196309 594762017 4529
496121037 238828086 365786
798232324 29858554 375121
776692215 119881968 41314
344714198 272569872 288048
950879300 596330 229439
845106481 10231574 712340
906792313 90251805 266175
50899642 266251734 280059
534059687 118393385 620106
823197239 59414211 605601
782599405 106224153 507116
872002250 46497937 508508
479109909 261437773 178310
62376064 310357759 480518
796725174 130467922 572316
778678589 104548282 233180
763880342 234354528 331013
889848860 38056130 306599
414403125 483259756 211553
275221051 162071406 908291
991078782 8792968 861485
60184339 354185853 174893
381099735 329296134 847525
329337585 406199395 958322
345249511 572601680 962892
205313907 719865941 654084
878056713 23451128 728337
628749180 322784147 891534
85620205 770360320 832676
164808472 115746570 575130
651097983 33497799 440535
950593506 13695100 336018
158341317 649177581 755287
127915955 27600907 633968
48510685 897088370 836942
328697587 219143610 261976
230621295 607357139 441609
319957874 136068915 641316
388936995 78117466 148532
761116599 118723267 875918
371120866 447431750 27329
318030717 644705752 211767
168179345 530406421 817108
587342196 221712703 550613
672352603 215389119 757258
399079150 21531674 215788
754616744 40804278 865956
353111581 534569371 381388
870041011 103911138 730174
755932617 183460520 788673
804846821 41018020 124595
409885856 522101514 625348
368129825 369759455 70096
811824921 95489863 649353
304363995 669228321 198053
849552974 87763786 991435
18499237 294431382 740037
520013319 269700196 830643
190605123 306143262 496470
854226933 100953797 916785
856901880 79063049 296579
888916259 50108519 532688
619195820 60727977 620882
216308074 29345768 218715
582602625 346257186 801030
792684855 168054436 68399
32203041 765254066 49406
728574717 137878553 234081
399100017 489850142 479169
41622663 939889756 698615
636085265 231359449 390159
280539556 460600328 507196
946518061 16187560 660613
971456112 11662541 619101
326787118 174772868 274392
737904004 115513478 959552
838027729 111408058 985643
406527799 219959025 935580
110158899 839699423 190747
553888624 446049330 703927
416923587 245716196 982861
507716749 136030472 1432
968576192 491728 844018
58408027 141015607 599362
880185835 98560301 255698
809748002 34923603 305085
674289131 190113720 150900
779167788 59357689 406420
896656336 29869294 421722
225126128 458222324 345945
41267221 432413647 720688
99617917 376651590 64036
363589187 254481557 863863
153450588 145638127 359600
327651434 188278704 107776
822781251 109747483 702169
384366491 123800500 559679
469939812 252219443 367652
349737732 14209913 114559
598974090 364688735 657701
809799571 119082542 992299
291895048 331359354 374606
162813391 23548174 530036
423836757 452149145 818958
985505704 10469980 751327
71080879 260442476 730580
162260308 25924407 165432
744661481 46289906 573863
55336579 136684733 452878
930710794 32931065 206980
881203638 28337708 411602
761904704 224011487 668375
561010697 129623870 642870
945260497 22790479 74851
535089045 146256879 194811
642959259 350521272 125619
882479139 98024755 521796
788793447 38254199 241972
503037323 161597197 279685
508935697 45018491 874126
429914773 158858394 577831
802720075 79138862 276298
967529897 12950123 820462
103523502 447779297 148189
486200693 203904480 243946
322638643 112942152 295370
315270756 561314890 675984
903635008 83909698 743729
420791493 123713162 726502
186775987 636095125 229745
676043379 33675537 341873
647276409 133857560 674030
404054453 304596096 682623
831401413 133434345 106278
941551664 52315421 163406
177661400 115926068 195459
510255718 417661258 72512
959947827 33023900 142128
928381032 27400567 90710
72934715 377872787 653918
404593035 193929599 159726
897076523 78886886 213660
482766717 76348260 721701
377167649 279955121 542827
838809814 158953074 24888
669105230 97932249 55674
95711073 813905992 457679
900946374 77395769 177867
343740564 627446583 803081
58968807 761841678 237375
262468088 381065403 327308
358504419 473374773 114488
328439951 361909997 373036
335330687 603852493 326059
919324117 35643346 604195
265988160 384921277 510427
456953433 89219726 394634
850647674 19562254 528254
419396670 181265444 348810
9168958 300344225 20452
642093479 69194968 457603
622592829 289279066 897432
186632747 798372071 940627
284742670 299082808 483684
836527588 145309460 153502
903248431 2249510 394838
666441024 38519404 64656
340576841 398782737 313638
53495936 785090738 218881
73412609 380099861 318612
370477548 328118215 275722
273260044 565444595 356187
300708524 519200619 628475
301788835 520920963 71413
765914797 112293723 426162
310259896 662509253 610942
206427184 430346446 355573
438680733 412013413 239525
321047553 441196095 266428
451909726 321269662 924348
110084520 718512483 446223
729392195 268089458 451806
469198959 179861042 206869
167744327 818473923 628800
765612172 80513915 688801
546044938 429290275 862789
286819861 577063797 918549
363279763 146279990 819528
503660357 286551884 867437
265367838 477434634 180173
35359266 830463189 790093
33286962 615952968 13017
555382085 96401425 405041
744225991 138916115 570445
143304744 753948591 914698
835503321 25340464 608154
294011418 134623332 171076
936635117 2280209 206113
16007314 501973757 952408
371864975 66719538 909236
853134670 30334932 215172
99056706 420808378 153536
270175194 554775502 292758
541677240 127421022 416584
840035127 75297228 525350
74605908 5482186 172993
129440940 462609193 722897
487798018 49829520 519234
915394924 26177821 17388
414044658 507245710 833151
56222157 640502146 911196
55672586 369519908 575402
280544805 235996806 173387
433244122 387834057 20359
505296692 419936957 61281
572261694 39001606 530852
765254328 39022567 770300
763186739 182210378 589362
671663486 158579997 524465
812670195 94199223 84961
15115497 949525715 40695
867876885 83283346 212304
752049202 75506633 834693
900507591 50830064 46444
953512831 45285446 854890
600096091 344949751 99234
265853669 486804911 928162
631393225 18316630 736950
258130946 541929106 941876
132421344 320286016 37038
642547640 282570582 185511
779729419 31706428 302300
10003596 912282734 698070
733223270 192159531 824948
227571733 658747558 314494
367687082 130674143 325425
654526877 48986360 952968
28167516 106318931 628903
322358251 57822865 175714
322525287 568381917 323245
460692550 465607955 165747
136429458 102588669 45507
781408202 176568490 385389
867398471 94194903 534823
844524011 59214347 856192
970959134 14564999 331682
639037028 110337199 737438
133128468 94057872 313382
316925546 538844218 83075
625553335 199434566 796551
483116921 364959304 695579
631717314 44799746 247819
360202552 311878185 46837
41409160 88654915 332090
384177425 319519591 974528
242949841 725820007 572811
934855549 55451615 184680
243544509 566192937 240355
622371643 147434520 654096
674611656 246802626 321707
151138744 90766284 192372
912771486 1599749 536399
697494304 278366881 144257
584428901 270576526 469614
86936816 86895010 802891
527286558 25276338 969416
532362270 194155128 3231
380427498 365430262 345314
630858186 283309190 703163
596949776 7496774 559279
421432359 213709831 606081
199256453 552512443 158216
924296922 33432150 854549
923515540 17575019 726690
910654669 9601284 243693
560161427 349600747 196544
121571919 457106176 695523
325512619 505060796 279921
852687160 146124488 940411
594697306 17522362 325024
181653647 510963837 136816
330559337 647571398 851932
910685561 81503616 499317
159112594 816806504 162582
537750150 168701350 463603
747695696 63669850 433941
467789376 206803060 318666
228039956 329931118 750090
738385772 155302099 373630
910230126 49499581 956595
981783892 9574866 23918
221101432 561150292 672351
786722850 135699747 409616
881353446 112255955 676900
508813081 424166734 110685
293077229 303356698 632514
514412923 358394617 793212
379505087 439269180 789831
424780119 522312415 376635
866373442 52106612 564742
292061807 175756404 734974
258932168 456704440 855589
577866702 289803221 543270
393537258 62704111 280105
408712424 125435417 654327
180804218 14997565 340630
232419099 624041335 406291
713359982 161650893 354072
400957559 506341478 732988
312722532 297026321 497899
109956600 147332684 410269
338261595 498980851 586161
142645461 138657229 419965
215253755 760453979 984853
298352110 112285807 785213
465573390 274682672 21371
484581478 324192936 334730
448142326 406556858 998758
57772485 229796325 23771
78598958 705868939 529171
723688096 116926971 5221
323861083 8340470 380385
215537274 751323126 57268
989637443 3953124 139820
239873506 591553647 971644
787797592 52459343 935633
221575231 400688507 180570
760726207 174919710 251898
978861976 2061209 213592
703725327 126154606 80975
398247535 202900796 382957
784650552 212624163 303829
724371442 124344520 134625
510581972 138117113 45072
994694125 2036398 115958
491642798 264007586 305043
966708820 5836916 865525
354997747 535643765 888948
173082314 811844189 457727
822191218 113929411 521755
542021117 26218277 537505
655030871 124322498 480201
210667061 411366384 69916
296004241 402996095 416970
409611907 7791688 137505
407327681 522718797 78715
436547484 413044759 59683
924805122 49018455 612880
603012304 85646944 104263
387524767 17603922 459577
333655812 453525430 934159
917992174 35274673 316092
501309364 436381404 691062
87357426 167595320 408343
529553460 134968091 944914
484643996 300854092 897199
111661355 291843535 14933
280466521 532313871 133135
763812031 59841845 926359
715007415 2855996 488597
852445784 128262071 181406
268596700 624320712 520784
567641111 247922534 440533
837027599 81771481 76643
34594405 65007338 259939
606282372 39150298 31887
994006048 2677854 138158
917186930 67848868 834351
93525232 266838198 284489
728221620 54343101 802670
151932168 643062588 646396
344810224 329449417 604216
330903118 297633145 621948
597433412 110894036 949161
341053643 151971193 202051
34049172 309057136 664914
163423959 19681514 167763
948417331 41648543 754194
683523515 46737616 558237
607695940 50215139 629801
173187879 35562369 226489
953191757 6840981 194196
966557029 7497578 41410
454533963 464014055 511897
938512075 16849056 272024
130936845 382513755 296353
568719635 474538 349961
977975124 1903395 566473
655883711 61886132 786274
806047698 62345648 362688
848617690 81189063 332089
129997972 176916778 849409
513454110 129671459 436472
373790293 551853208 776454
445563884 206835490 891784
714175980 137194390 843419
376905235 289224104 20467
764251519 200927797 25878
925179384 14200410 560888
720783605 239051243 504118
240661628 9221346 602812
46697361 768148318 598364
772363383 190296589 157460
941408343 14658804 173078
403899228 359409887 764210
273163722 275297661 348952
287174641 362359252 934764
106093809 385921348 512086
419498607 336370081 202012
632176974 321695465 511746
904007584 71403250 293601
892915423 10782046 925353
2052346 481755536 342050
89547384 182578479 617688
154219626 719753559 439649
110212299 163735217 279591
613257745 349833662 416854
128287655 421243969 776305
162301409 283015089 212148
872237817 116321520 98834
885351027 110646052 53294
597926586 83141063 493630
572168839 332246847 917787
417730333 410098367 809457
859432476 113463271 546775
539238928 328821660 14480
26526561 904935662 153157
765158241 51036327 919321
657613895 169285078 468587
53366755 106638171 431081
544254170 455171646 91373
622245818 127829644 699067
344979022 446995068 788438
336271278 405974397 75592
900149337 70872322 75373
798091986 30935171 39541
762690692 15445222 318580
308318980 8143164 825052
558254545 45796654 993985
479820423 314078208 74341
1231488 398106121 587025
509466109 218761315 200847
974866378 9515342 205938
782308411 5715930 77758
227270486 602715517 470814
625934621 154299599 836916
521655809 18043571 152107
588208896 93822744 981636
999154392 714849 858408
634015654 138704206 628823
4196632 975542143 911165
695130185 45779636 737472
958287060 14763338 109822
662365584 14931987 971727
61007224 28406462 90237
253704825 454751450 604036
817025724 49720921 913521
553604521 62479511 340393
628414764 307153717 332536
98424461 291273429 487829
423650367 141277359 300623
675690172 125182371 131055
542173048 271089701 686483
779782054 213369375 1264
488464192 398923261 250006
972119895 13531737 465644
687161934 296936929 372776
373287089 418178342 556347
152957493 768793575 989654
656593024 41663176 768526
798084278 30176665 91708
763385154 194540074 236065
80627509 722982686 943515
389734164 155459026 732131
194931494 689476485 395083
751161769 151166126 648009
602015642 336158324 393856
354473579 483050188 311799
844272514 5364668 28813
944410799 6048947 466262
428332939 552369074 944681
979920078 11339678 191993
531906577 62481441 101982
796587174 127646949 547094
314168249 531649549 846031
707130517 208638668 160156
969729335 12133614 164294
862931699 3953720 788436
808657652 142245096 585042
736344611 232617777 440202
966670193 10741113 903893
934315326 5108910 332099
53569123 128320735 619765
8346875 970800136 120156
