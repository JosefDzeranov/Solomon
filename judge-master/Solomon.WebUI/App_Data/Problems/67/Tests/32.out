28867610
47
397 9 87 651 479 548 304 846 6 883 767 544 365 790 174 252 711 51 832 245 673 899 80 805 431 206 873 330 97 451 933 218 53 611 419 844 263 925 450 48 782 339 833 229 769 92 894 